EIPS/eip-2535.md

```diff
@@ -24,13 +24,13 @@ New terminology from the diamond industry.
 A diamond is a proxy contract that supports using multiple logic/delegate contracts at the same time. In this standard the term for logic/delegate contract is facet. A diamond can have many facets. Which facet is used depends on which function is called. Each facet supplies one or more functions.
 
 A diamond provides:
-
-1. A way to add, replace and remove multiple functions atomically (in the same transaction).
-2. An event that shows what functions are added, replaced and removed.
-3. A way to look at a diamond to see its functions.
-4. Solves the 24KB maximum contract size limitation. Diamonds can be any size.
-5. Enables zero, partial or full diamond immutability as desired, and when desired.
-6. Designed for tooling and user-interface software.
+1. A way to create a set of contracts (facets) under a single address that are independent from each other but can share functionality and contract storage.
+2. A way to add, replace and remove multiple external functions atomically (in the same transaction).
+3. An event that shows what functions are added, replaced and removed.
+4. A way to look at a diamond to see its facets and functions.
+5. Solves the 24KB maximum contract size limitation. Diamonds can be any size.
+6. Enables zero, partial or full diamond immutability as desired, and when desired.
+7. Designed for tooling and user-interface software.
```

Comment:
You can just use the number `1` for each line item so you don't have to maintain the numbers manually.
```
1. First.
1. Second.
1. Third.
```
> 1. First.
> 1. Second.
> 1. Third.

Reply:
Wow, that is a great tip!  Thanks @MicahZoltu